### ISSUE TITLE
Warn of payments to exchanges without memo

### ISSUE THREAD

**Issue by ghost** [stellar/account-viewer#48]
We see so many users forgetting to set the memo when sending funds to exchanges like Bittrex and Kraken that we should consider displaying a warning when they try to do so. I've noticed  stellarchain, stellar dashboard, and stellar expert all identify these exchanges (perhaps by home domain? or is this hard coded?).

**Comment by ghost** [comment]
I see dashboard has a [known_accounts.js](https://github.com/stellar/dashboard/blob/master/src/common/known_accounts.js). Is this something that is useful to maintain separately and host on a CDN? As more projects need to keep a registry of accounts it's maybe a good idea to have only a single source to keep up to date.

**Comment by bartekn** [comment]
Yes, we will extract `known_accounts.js` and maybe create a web service around it. I think that PR adding this feature (with static list of exchanges in the beginning) will be approved.
